test/SourceKit/CodeComplete/complete_docbrief_package.swift

```diff
@@ -0,0 +1,50 @@
+// BEGIN Module.swift
+package protocol P {
+  /// This is a doc comment of P.foo
+  ///
+  /// Do whatever.
+  func foo()
+}
+
+package struct S: P {
+  public init() {}
+  public func foo() {}
+}
+
+// BEGIN User.swift
+package import DocBriefTest
+
+func test() {
+  S().foo()
+}
+
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/Modules)
+// RUN: %{python} %utils/split_file.py -o %t %s
+
+// RUN: %target-swift-frontend \
+// RUN:     -emit-module \
+// RUN:     -module-name DocBriefTest \
+// RUN:     -package-name DocPackage \
+// RUN:     -emit-module-path %t/Modules/DocBriefTest.swiftmodule \
+// RUN:     -emit-module-source-info-path %t/Modules/DocBriefTest.swiftsourceinfo \
+// RUN:     %t/Module.swift
+
+// RUN: %sourcekitd-test -req=complete -pos=4:7 %t/User.swift -- %t/User.swift -I %t/Modules -target %target-triple -module-name DocBriefUser -package-name DocPackage -enable-experimental-feature AccessLevelOnImport | %FileCheck %s -check-prefix=CHECK
+
+// CHECK: {
+// CHECK:   key.results: [
+// CHECK-NEXT:     {
+// CHECK-NEXT:       key.kind: source.lang.swift.decl.function.method.instance,
```

Comment:
I'd prefer only checking the things we're actually testing here, just name + doc is probably enough (same for the other tests).

Reply:
Updated to drop some of the expectations, I would like to keep the expectations between the name and doc comment though, as that lets us continue to use `CHECK-NEXT`, instead of needing to put in a `CHECK-NOT: }`, which reads weirdly IMO.